fhir-install/docker/deploySchemaAndTenant.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+###############################################################################
+# (C) Copyright IBM Corp. 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################################
+set -ex
+
+java -jar volumes/schema/fhir-persistence-schema-4.0.1-SNAPSHOT-cli.jar \
```

Comment:
If we increment the version, it could be a pain to pick this up. 
Maybe pickup version somewhere else?

Reply:
i think we could use a `*` in the jar name if you're comfortable with that

Reply:
schema-*-SNAPSHOT?

Reply:
I think it makes sense - and it's controlled

Reply:
I was thinking `fhir-persistence-schema-*-cli.jar`, but if you want to restrict it to SNAPSHOTs just let me know.

Reply:
good point... I think you have the right approach.